testsuite/unittests/test_iaf_psc_exp_ps_lossless.sli

```diff
@@ -263,13 +263,13 @@ dc_gen nrn_spike 1. 1. Connect
 
 %read out spike response of nrns:
 
-/spike_detector Create /det_nospike Set
+/spike_recorder Create /det_nospike Set
 nrn_nospike det_nospike Connect
 
-/spike_detector Create /det_missingspike Set
+/spike_recorder Create /det_missingspike Set
 nrn_missingspike det_missingspike Connect
 
-/spike_detector Create /det_spike Set
+/spike_recorder Create /det_spike Set
```

Comment:
These should all use `rec_` instead of `det` as well

Reply:
Yep.